src/analyses/libraryFunctions.ml

```diff
@@ -64,6 +64,7 @@ let c_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("__builtin_strcmp", special [__ "s1" [r]; __ "s2" [r]] @@ fun s1 s2 -> Strcmp { s1; s2; n = None; });
     ("strncmp", special [__ "s1" [r]; __ "s2" [r]; __ "n" []] @@ fun s1 s2 n -> Strcmp { s1; s2; n = Some n; });
     ("malloc", special [__ "size" []] @@ fun size -> Malloc size);
+    ("alloca", special [__ "size" []] @@ fun size -> Malloc size); (* TODO: Maybe define a new special type [Alloca], just like [Malloc]? *)
```

Comment:
Yes, having a special type here makes sense!

Reply:
Added `Alloca` type in https://github.com/goblint/analyzer/pull/1179/commits/aa97af86c23911bd05822149a7d9c9fc4f819ae2. I also moved `__builtin_alloca`, s.t. it's now classified only as `Alloca` and not as `Malloc` anymore: https://github.com/goblint/analyzer/pull/1179/commits/6e9bb2f121114a9c91bf1bb19557bbcfd0d3ffca